lib/easymon/checks/http_check.rb

```diff
@@ -19,15 +19,22 @@ def check
     end
     
     private
-      def http_up?(config_url)
-        response = RestClient::Request.execute(
-                                      :method => :head, 
-                                      :url => config_url,
-                                      :timeout => 5,
-                                      :open_timeout => 5)
-        true
+      def http_up?(url)
+        http_head(url).is_a?(Net::HTTPSuccess)
       rescue Exception
         false
       end
+
+      def http_head(url)
+        uri = URI.parse(url)
+
+        http = Net::HTTP.new(uri.host, uri.port)
+        http.use_ssl = uri.is_a?(URI::HTTPS)
+        http.verify_mode = OpenSSL::SSL::VERIFY_NONE
```

Comment:
Why `VERIFY_NONE` ? Shouldn't we want a strong verification of our own SSL certs?